src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf

```diff
@@ -292,11 +292,47 @@
             </trans-unit>
             <trans-unit id="76">
                 <source>The image is landscape oriented ({{ width }}x{{ height }}px). Landscape oriented images are not allowed.</source>
-                <target>Slika je orijentacije pejzaža ({{ width }}x{{ height }}px). Pejzažna orijentacija slika nije dozvoljena.</target>
+                <target>Slika je pejzažno orijentisana ({{ width }}x{{ height }}px). Pejzažna orijentisane slike nisu dozvoljene.</target>
             </trans-unit>
             <trans-unit id="77">
                 <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
-                <target>Slika je orijantacije portreta ({{ width }}x{{ height }}px). Portretna orijentacija slika nije dozvoljena.</target>
+                <target>Slika je portretno orijentisana ({{ width }}x{{ height }}px). Portretno orijentisane slike nisu dozvoljene.</target>
+            </trans-unit>
+            <trans-unit id="78">
+                <source>An empty file is not allowed.</source>
+                <target>Prazna datoteka nije dozvoljena.</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Nije moguće odrediti poslužitelja.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Vrednost se ne poklapa sa očekivanim {{ charset }} setom karaktera.</target>
+            </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Ovo nije validan BIC.</target>
```

Comment:
don't we want to add something that'd mean "Business Identifier Code"? BIC is clear enough? (same below)

Reply:
Don't know if we have a word for BIC, maybe we can literally translate it